### ISSUE TITLE
ncar_boundary_scaling algorithm results in inconsistent background viscosities

### ISSUE THREAD

**Issue by russfiedler** [mom-ocean/MOM5#282]
There's already a note in the code partly adressing this issue but it skirts the main problem.

The background viscosities `aiso_back` and `aaniso_back`  are on the velocity points. When using the NCAR algorithm  ocean_bihgen_friction` and `ocean_lapgen_friction_mod` in this results in an inconsistency in the background values for (i,nj,:) and (ni-i,nj,:) which are actually the same point. 

For `i<ni/2` distances along the fold  are measured from Russia and `i>n/2` distances are measured from Canada. In short, we have 2 different viscosities for the same point!

I suggest a warning be issued is a user tries to use this algorithm and suggest that they calculate the background scaling correctly offline.

**Comment by aekiss** [comment]
Also the `ncar_boundary_scaling` boundary scaling is supposed to ensure western boundary currents are wide enough to resolve, so it should be on the eastern side of land cells. But is actually on the i-maximum side, which is not the same thing in the tripolar cap region. Perhaps it should simply be disabled in the tripolar cap region?

**Comment by StephenGriffies** [comment]
Good catch..  Unless someone wishes to work through the algorithm for the bipolar region, then it might just be best to remove it north of 65N.  However, that should be tested to ensure no major negative changes appear.
